src/motion/actuators/GenericActuator.hx

```diff
@@ -240,7 +250,7 @@ class GenericActuator<T> implements IGenericActuator {
 		
 		if (duration == 0) {
 			
-			complete ();
+			complete (true);
```

Comment:
The argument is `true` by default. While I personally prefer explicit argument passing, I still don't get why you decided to change this :)